runtime-kernel/broadcom-wl/autobuild/defines

```diff
@@ -0,0 +1,7 @@
+PKGNAME=broadcom-wl
+PKGSEC=non-free/kernel
+PKGDES="Broadcom 802.11 Linux STA wireless driver"
+PKGDEP="dkms"
+
+FAIL_ARCH="!(amd64)"
```

Comment:
Hmm. Are we sure about that? I'll do a quick test on LoongArch.

Reply:
Broadcom ships binary blob for i386 and amd64. The 64-bit goes to /usr/lib and linked to kernel module. Unfortunately...

Reply:
I haven't checked what are the symbols there, it could be just the firmware but it's big.

Reply:
Yep. Not working.